src/OpenLoco/src/Vehicles/Vehicle.h

```diff
@@ -73,6 +73,16 @@ namespace OpenLoco::Vehicles
     };
     OPENLOCO_ENABLE_ENUM_OPERATORS(SoundFlags);
 
+    enum UpdateVar1136114Flags : uint32_t
```

Comment:
Should make this an enum class and add `OPENLOCO_ENABLE_ENUM_OPERATORS` as its a set of flags. You'll then need to update all the comparisons to compare against `none`. Its slightly painful to do so we often add a helper function for this so maybe something like `hasUpdateVar1136114Flags(UpdateVar1136114Flags flags)` eventually i want to rework how this global works and not pass it round like it is but for much further in the future.